### PR TITLE
jeez steelpoint I know your snowflake role is important but >20 cost

### PR DESCRIPTION
**Pull request by Iamgoofball**
\>testmerge dorfs
\>wonder why only two ruins exist in all of lavaland
\>20 cost for a fucking security officer ripoff

### CODE REVIEW

**Comment by Kromgar** [comment]
So there can be less ruins if high point ones spawn?

**Comment by optimumtact** [comment]
@Kromgar it uses up more of the budget yeah

**Comment by kevinz000** [comment]
please make it even lower jesus fucking christ

**Comment by kevinz000** [comment]
god fuc king damnit @Steelpoint

**Comment by lzimann** [comment]
fix the title please

**Comment by FantasticFwoosh** [comment]
Dunno about you, but it seems spot on actually for a title.

**Comment by ike709** [comment]
The title is accurate.

File: code/datums/ruins/lavaland.dm [hunk 1]
```diff
@@ -190,4 +190,4 @@
 	description = "A Marine	 Transport that has crashed, with one sole survivor"
```
Comment (KorPhaeron) [inline code/datums/ruins/lavaland.dm]: As opposed to multiple sole survivors

**Comment by Steelpoint** [comment]
I made the cost consistent with other high spawn roles, I thought that was the standard thing to do and I thought the cost simply made it rarer for ruins to spawn. There is no documentation on procedure on how to set costs so I apologise.

**Comment by Iamgoofball** [comment]
It's cool, we fixed it.

**Pull request merged**